evm_arithmetization/src/cpu/kernel/constants/mod.rs

```diff
@@ -394,20 +396,26 @@ pub mod cancun_constants {
         hex!("000000000000000000000000000000001666c54b0a32529503432fcae0181b4bef79de09fc63671fda5ed1ba9bfa07899495346f3d7ac9cd23048ef30d0a154f"), // y_im
```

Comment:
Unrelated to this PR but what is `y_im`?

Reply:
Imaginary part of Y (as Fp2 element)